app/views/state_file/questions/esign_declaration/edit.html.erb

```diff
@@ -3,10 +3,27 @@
 
 <% content_for :card do %>
   <h1 class="h2"><%= title %></h1>
-  <%= current_state_code == "ny" ? t(".ny.body_html") : t(".az.body_html") %>
-  <%= form_with model: @form, url: { action: :update }, data: { efile_security_information: true, form_name: "state_file_esign_declaration_form" }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card" } do |f| %>
+  <% if current_state_code == "md" %>
+    <%= t(".md.body_html") %>
+  <% elsif current_state_code == "ny" %>
+    <%= t(".ny.body_html") %>
+  <% else %>
+    <%= t(".az.body_html") %>
+  <% end %>
```

Comment:
The use of state codes here is not ideal here, but it's not part of this ticket to fix that